`make htmldocs` is broken again and this is a temporary fix for that. the html
docs may be malformed but at least it builds without error ¯\_(ツ)_/¯

source: https://bugs.archlinux.org/task/66156#comment188161
upstream issue: https://github.com/sphinx-doc/sphinx/issues/7421

this should be removed once the issue is resolved. note that there will be a lot
more errors/warnings than usual due to ripping such an integral part out of
the docs parser.

diff --git i/Documentation/conf.py w/Documentation/conf.py
index 3c7bdf4cd31f..9a0ced58a3e9 100644
--- i/Documentation/conf.py
+++ w/Documentation/conf.py
@@ -36,7 +36,7 @@ needs_sphinx = '1.3'
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ['kerneldoc', 'rstFlatTable', 'kernel_include', 'cdomain',
+extensions = ['kerneldoc', 'rstFlatTable', 'kernel_include',
               'kfigure', 'sphinx.ext.ifconfig', 'automarkup',
               'maintainers_include']
 
